source/errors.ts

```diff
@@ -45,41 +46,42 @@ export class ReadError extends GotError {
 }
 
 export class ParseError extends GotError {
-	readonly response: Response;
+	readonly response!: Response;
```

Comment:
Typescript complains that response is not initialized and definitely assigned in the constructor, because it won't pickup `Object.defineProperty` calls as a property assignment. I added this since from the source it is known that this will be defined within the constructor.

Reply:
Alright. Thanks for explaining.